install.sh

```diff
@@ -3920,6 +3997,21 @@ setup_shared_hook_repositories() {
     fi
 }
 
+############################################################
+# Finalizes the installation.
+#   Sets the global installation directory.
+#
+# Returns:
+#   None
+############################################################
+finalize_installation() {
```

Comment:
I'd set this up early, after we parsed the arguments, we know if we'll use the default or the prefix, so can be set right after that point.

Reply:
I set it at the end with the intention, when the installation fully worked -> its fine to set it. 
hm...